frame/erc20/src/lib.rs

```diff
@@ -81,20 +81,20 @@ decl_module! {
         }
 
         /// Transfers some amount of the native token to some recipient on a (whitelisted) destination chain.
-        #[weight = 195_000_000]
-        pub fn transfer_native(origin, amount: BalanceOf<T>, recipient: Vec<u8>, dest_id: bridge::ChainId) -> DispatchResult {
-            let source = ensure_signed(origin)?;
-            ensure!(<bridge::Module<T>>::chain_whitelisted(dest_id), Error::<T>::InvalidTransfer);
-            let bridge_id = <bridge::Module<T>>::account_id();
-            T::Currency::transfer(&source, &bridge_id, amount.into(), AllowDeath)?;
-
-            let resource_id = T::NativeTokenId::get();
-			let number_amount: u128 = amount.saturated_into();
-
-			// <bridge::Module<T>>::transfer_fungible(dest_id, resource_id, recipient, U256::from(amount.saturated_into()))
-            <bridge::Module<T>>::transfer_fungible(dest_id, resource_id, recipient, U256::from(number_amount))
-			// <bridge::Module<T>>::transfer_fungible(dest_id, resource_id, recipient, U256::from(UniqueSaturatedInto::<u128>::unique_saturated_into(amount)))
-        }
+        // #[weight = 195_000_000]
```

Comment:
Please, use [block comment](https://doc.rust-lang.org/reference/comments.html).

Reply:
Updated to use block comment